_posts/2020-04-28-history-of-bel.md

```diff
@@ -35,30 +35,72 @@ community. And so, we being on a dark and stormy night in 2001....
+
+**November 2004** Genstruct and Pfizer extend and expand their partnership, applying the Genstruct platform to various R&D and Toxicology programs throughout Pfizer. (https://www.genomeweb.com/archive/genstruct-pfizer-expand-research-partnership#.X3jczJNKh24)
 
 ### 2006
 
-**Sometime in 2006** Selventa breaks even and has positive cash flow ([ref](https://www.linkedin.com/in/keithelliston/)).
-Keith Elliston's LinkedIn profile claims that it was profitable in 2008 and 2009.
+**August 2006** Genstruct and GSK extend their partnership to apply the Genstruct platform to define compound mechanisms of action in Oncology (https://www.fdanews.com/articles/61568-genstruct-extends-collaboration-in-cancer-with-glaxosmithkline)
+
+**Sometime in 2006** Gensruct reaches cashflow break even ([ref](https://www.linkedin.com/in/keithelliston/)).
+Keith Elliston's LinkedIn profile claims that it was cash flow positive in 2008 and 2009.
+
+### 2008
+
+**April 30, 2008** Genstruct and Sirtris win the BioIT World Best Practices Award for their work using the Genstruct Platform to identify the mechanisms of action for the Sirtris Sirt1 activators.
+Sirtris was acquired by GSK to further develop its Sirt1 activators.
+
+**July 2008** Genstruct begins a collaboration with Manuel Pietsch and his group on the use of the Genstruct technology to assess the stength and extent of toxicity using network analysis
+
+### 2009
+
+**2009** Genstruct develops the "Network Perturbation Amplitude" algorithm, based on early work done by Jim Watters on pathway expression activation.
+
+**September 2009** Board appoints Chris Varma from Flagship Ventures as Executive Chairman.
 
 ### 2010
 
-**May 17th, 2010** Selventa [raises $500K](https://www.crunchbase.com/funding_round/selventa-series-unknown--a7424e79#section-overview)
-in a Venture Round. This is the second of three rounds of funding, the last of which will occur in late 2011.
+**Jan 2010** Keith Elliston resigns from the company, [Chris Varma](https://www.linkedin.com/in/chris-varma-59828a4/) named CEO (though this is missing from his LinkedIn profile).
 
-**November 10th, 2010** Selventa rebrands from GenStruct, Inc. to Selventa ([ref](https://www.businesswire.com/news/home/20101130005216/en/Genstruct-Selventa-Company-Launches-Re-branding-Effort))
+**May 17th, 2010** Genstruct [raises $500K](https://www.crunchbase.com/funding_round/selventa-series-unknown--a7424e79#section-overview)
+in a Venture Round. This is the second of three rounds of funding, the last of which will occur in late 2011. 
 
-**December 8th, 2010** Selventa replaces Keith Elliston as CEO with David de Graaf ([ref](https://www.biospace.com/article/releases/-b-selventa-b-names-b-david-de-graaf-b-as-president-and-chief-executive-officer)).
```

Comment:
Somehow this linked changed to have a strange trailing dash: https://www.biospace.com/article/releases/-b-selventa-b-names-b-david-de-graaf-b-as-president-and-chief-executive-officer-/. I'll add it back in